WalletWasabi.Bench/TransactionProcessorBench.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using NBitcoin;
+using WalletWasabi.KeyManagement;
+using WalletWasabi.Models;
+using WalletWasabi.Services;
+using System.IO;
+using WalletWasabi.Helpers;
+using System.Runtime.CompilerServices;
+using NBitcoin.Protocol;
+using WalletWasabi.Stores;
+using System.Net;
+
+namespace WalletWasabi.Bench
+{
+	[ClrJob(baseline: true), CoreJob]
+	[RPlotExporter, RankColumn]
+	public class TransactionProcessorBench
+	{
+		private TransactionProcessor _tp;
+		private List<SmartTransaction> _txs;
+
+		[Params(100, 1_000, 10_000)]
+		public int TRANSACTIONS;
```

Comment:
This field should be private
`private int _transactions`

Reply:
No. It is expected to be public in order to allow the benchmarking framework to inject the values.